Packs/CommonScriptsV2/Scripts/GetListDatawithKeyword/GetListDatawithKeyword.py

```diff
@@ -0,0 +1,51 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+from typing import Dict, Any
+import traceback
+
+result_data=[]
+
+def get_data(key_word, json_data):
```

Comment:
I think we might be too specific here..
Let's consider turning this transformation to something that gets a List that can contain either dicts or lists (or both) and retrieve all values inside that nested objects that contain the given keyword (make sure it's a set so we don't get duplications)
Wdyt?

Reply:
Hi Yuval, I have made this script to make use as transformer and i forgot to give the tags: transformer

Reply:
I think thats a good idea, i will update